.appveyor.yml

```diff
@@ -55,15 +59,22 @@ install:
   - cmd: if %compiler%==msvc2015 ( call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %arch% )
   - cmd: if %compiler%==msvc2017 ( call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat" -arch=%arch% )
   - cmd: if %compiler%==msys2-mingw (if %arch%==x86 (set "PATH=C:\msys64\mingw32\bin;%PATH%") else (set "PATH=C:\msys64\mingw64\bin;%PATH%"))
+  - cmd: if %compiler%==cygwin ( call ci\appveyor-install.bat )
```

Comment:
This doesn't install Cygwin, just packages not in the AppVeyor Cygwin install needed to test meson: ninja,gcc-objc,gcc-objc++,libglib2.0-devel,zlib-devel.  Dropping the frameworks tests reduced this list a bit, so hopefully that makes it a little faster.